src/xpk/parser/workload.py

```diff
@@ -601,9 +633,9 @@ def add_shared_workload_create_optional_arguments(args_parsers):
         ),
     )
     custom_parser.add_argument(
-        '--remote-python-sidecar-image',
+        '--colocated-python-sidecar-image',
```

Comment:
We need to adjust the name in a few more places. If you're able to can you make the changes in this PR?

https://github.com/AI-Hypercomputer/xpk/pull/451/files

Reply:
The sidecar image function is not needed anymore as PathwaysJob builds the container. Please note.

Reply:
Got it, thank you! I'll also check in the equivalent change in MaxText once this PR is in

https://github.com/AI-Hypercomputer/maxtext/pull/1532